packages/init/src/index.js

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env node
 /* eslint no-console: 0 */
+/* eslint-disable camelcase */
```

Comment:
Just thinking out load here, but is this ESLint exception this because of` PROJECT_API_URL`?  I guess the uppercase isn't really a big deal for this variable is it?  It seems we use a lot of strings as constant "hints" throughout our code (`templateStandardName` also seems like one too? 🤔 ) but never capitalize them all either.

I wonder if we should
1. either just go all in on caps for any `const` that is a primitive (boolean, string, number)
1. just turn this ESLint rule off?

Reply:
Had the same thoughts.

Reply:
Meant to add that I'm ok with option no. 2